client/state/Makefile

```diff
@@ -5,6 +5,9 @@ BASE_DIR := $(NODE_BIN)/../..
 NODE_PATH := $(BASE_DIR)/client
 
 test:
-	@NODE_ENV=test NODE_PATH=$(NODE_PATH) $(MOCHA) --compilers jsx:babel/register,js:babel/register --reporter $(REPORTER)
+	@NODE_ENV=test NODE_PATH=$(NODE_PATH) $(MOCHA) --compilers jsx:babel/register,js:babel/register --reporter $(REPORTER) $(addprefix --grep ,$(GREP))
```

Comment:
> The only downside is we don't get the setup/cleanup from the index.js file.

That seems like a big downside? 

Maybe something to bake into a custom runner at the root? Only add files that match a glob, but always add the root, which has setup / teardown.